storage/innobase/xtrabackup/doc/source/advanced/page_tracking.rst

```diff
@@ -0,0 +1,111 @@
+  .. code-block:: bash
+
+     INSTALL COMPONENT "file://component_mysqlbackup";
+
+2. Check whether the mysqlbackup component is installed succesfuly:
+
+  .. code-block:: bash
+
+     SELECT COUNT(1) FROM mysql.component WHERE component_urn='file://component_mysqlbackup';
+
+Using the page tracking
+-----------------------
+
+1. mysqlbackup processes all the tracked pages in memory. The component must be started with enough memory. Otherwise, the component throws an error and exits. For more information see the `MySQL documentation <https://dev.mysql.com/doc/mysql-enterprise-backup/8.0/en/mysqlbackup.incremental.html#mysqlbackup.page-tracking.incremental>`__ .
+
+2. After the mysqlbackup component is loaded and active on the server, start page tracking:
+
+  .. code-block:: bash
+
+     SELECT mysqlbackup_page_track_set(true);
+
+3. When you enable page tracking, it creates a file under the server's datadir to collect data about changed pages. This file is growing until you stop the page tracking. If you stop the server and then restart it, page tracking will create a new file but will also keep the old one. The old file will continue growing until you stop the page tracking explicitly.
+
+To purge old page tracking data, do the following steps:
+
+
+  .. code-block:: bash
+
+     SELECT mysqlbackup_page_track_set(false);
+     SELECT mysqlbackup_page_track_purge_up_to(9223372036854775807);
+     /* Specify the LSN up to which you want to purge page tracking data. 9223372036854775807 is the highest possible LSN which purges all page tracking files.*/
+     SELECT mysqlbackup_page_track_set(true);
+   
+3. Enable the page tracking functionality for the full and incremental backups with the following option:  
+
+``--page-tracking``
+
+.. note::
+
+   To use the page tracking option, you must have the `Backup_ADMIN <https://docs.oracle.com/cd/E17952_01/mysql-enterprise-backup-8.0-en/mysqlbackup.privileges.html>`__ privileges.
+
+The option has the following benefits:
+
+* Resets page tracking to the start of the backup. This reset allows the next incremental backup to use page tracking.
+
+* Allows the use of page tracking for an incremental backup if the page tracking data is available from the backup's start checkpoint LSN.
+
+If the server page flushing rate is low or it uses a very big buffer pool, you can start page tracking before creating the first backup. Thus, you ensure that page tracking LSN is more than the checkpoint LSN of the server.
+
+4. To check the LSN value starting from which changed pages are tracked, use the following option:
+
+  .. code-block:: bash
+
+     SELECT mysqlbackup_page_track_get_start_lsn();
+
+After enabling the functionality, the next incremental backup finds changed pages using page tracking.
+
+Stop page tracking
+------------------
+
+If you need to stop page tracking, use the following option:
+
+  .. code-block:: bash
+
+     SELECT mysqlbackup_page_track_set(false);
+
+First backup using page tracking
```

Comment:
Move this to the beginning of the section. This is information I would need to know.